tests/foreman/ui/test_activationkey.py

```diff
@@ -1290,11 +1288,45 @@ def test_delete_manifest(self):
 
         @Assert: Deleting a manifest removes it from the Activation key
 
-        @Status: Manual
-
-        @BZ: 1078676
-
         """
+        # Upload manifest
+        manifest_path = manifests.clone()
+        entities.Organization(
```

Comment:
Will it affect organization somehow? That org doesn't have any manifests uploaded in setup section, so i thought we can safely upload and then delete the manifest in our test without any impact.